src/library/basetracktablemodel.cpp

```diff
@@ -545,6 +545,16 @@ QVariant BaseTrackTableModel::roleValue(
     }
     switch (role) {
     case Qt::ToolTipRole:
+        switch (field) {
+        case ColumnCache::COLUMN_LIBRARYTABLE_PREVIEW:
```

Comment:
COLUMN_LIBRARYTABLE_PREVIEW is redundant, same as kDataExportRole.

Why do we need to skip COLUMN_LIBRARYTABLE_RATING and COLUMN_LIBRARYTABLE_TIMESPLAYED explicitly instead of simply displaying their value? Then this additional switch/case could be removed, same as before.

Reply:
I did consider the tool-tip on top of the rating stars and the played checkbox as visual clutter. 

The use case of the tool-tip is to see the full string if the column is to small. 
I have just noticed that this is also possible for these columns.
So I will re-enable the tool-tips.

OK?

Reply:
I have reverted these lines and git ammend